rcl_action/src/rcl_action/action_server.c

```diff
@@ -228,6 +228,7 @@ rcl_action_server_init(
 rcl_ret_t
 rcl_action_server_fini(rcl_action_server_t * action_server, rcl_node_t * node)
 {
+
```

Comment:
beep boop extra newline :robot:

Reply:
hehe, I saw that and tried to hide it before anyone would notice